src/mrsimulator/tests/test_core.py

```diff
@@ -8,12 +8,19 @@
 # from typing import ClassVar
 
 from mrsimulator import Site, Isotopomer, Spectrum
-from mrsimulator.tests import mas_data, static_data
+
+# from mrsimulator.examples import mas_data, static_data
 
 
 # Test Site
 def test_direct_init_site():
-    Site(isotope="29Si", isotropic_chemical_shift=10)
+    the_site = Site(isotope="29Si", isotropic_chemical_shift=10)
+    assert the_site.isotope == "29Si"
+    assert the_site.isotropic_chemical_shift == 10
+    # assert the_site.property_units["isotropic_chemical_shift"] == "Hz"
```

Comment:
@shyamd Shouldn't the unit of isotropic_chemical_shift default to 'Hz'. The unit is not defined when site object is defined directly.

Reply:
@shyamd Are Site, Spectrum and Isotopomer class suppose to be public? I don't see how these objects can be used to directly setup, for instace, the following Site object.

```python
from mrsimulator import Site
a = Site(isotope='1H', isotropic_chemical_shift='2.1 ppm')
```

Reply:
They should definitely be public, otherwise there is no way to assemble them for use in the Simulator.